armi/reactor/tests/test_components.py

```diff
@@ -938,12 +938,12 @@ class TestHelix(TestShapedComponent):
         "id": 0.1,
     }
 
-    def test_getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
+    def test_getBoundingCircleOuterDiameter(self):
```

Comment:
Good catch.

This went through the review of #747 yesterday without any notice!